_includes/selling-points.html

```diff
@@ -0,0 +1,47 @@
+<section class="layout">
+  <section class="content content--large content--grid content--centered">
+    <article class="content--grid__item">
+      <img src="images/icon-ruby.svg" alt="Keep your rails workflow" />
+      <h4>
+        Keep your Rails workflow
+      </h4>
+      <p>
+        Serve both your Ember and Rails applications from a single process.
+      </p>
+    </article>
+    <article class="content--grid__item">
+      <img src="images/icon-cloud.svg" alt="Simplofy your Heroky deployments" />
```

Comment:
Also, I would argue we don't need the alt text here since the next piece of content is a heading with the same text, so a screen reader would read aloud the same text twice.